packages/fast-components-styles-msft/src/utilities/color/palette.ts

```diff
@@ -235,12 +243,14 @@ export function swatchByContrast(referenceColor: string | SwatchResolver) {
                      * we return the color
                      */
                     return (designSystem: DesignSystem): Swatch => {
-                        const color: Swatch =
-                            typeof referenceColor === "function"
-                                ? referenceColor(designSystem)
-                                : referenceColor;
-                        const sourcePalette: Palette = paletteResolver(designSystem);
```

Comment:
You're deleting length but still referencing it. I don't think you want to do that.

Reply:
Doh! I thought I caught that merge.